.github/CODEOWNERS

```diff
@@ -237,6 +237,7 @@
 /core/templates/services/exploration-html-formatter.service*.ts @kevintab95
 /core/templates/services/image-upload-helper.service*.ts @kevintab95
 /core/templates/services/validators.service*.ts @EricZLou
+/core/templates/components/save-pending-changes/ @heyimShivam
```

Comment:
Add this in alphabetic order i.e, around line 221. Also, assign it to a LaCe team member.

@kevintab95 Is it okay to assign you to this folder?

Reply:
Done

Reply:
assigned @kevintab95  for now if he wants to change his name then I will do that in the next commit. @DubeySandeep

Reply:
Done

Reply:
@kevintab95 PTAL!

Reply:
No concerns!

Reply:
Also, sorry for missing it earlier!

Reply:
@DubeySandeep PTAL